rules/macos/credential_access_kerberosdump_kcc.toml

```diff
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/08/14"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/14"
+
+[rule]
+author = ["Elastic"]
+description = "Detects the use of kerberos cache manipulation kcc utility to dump cached kerberos tickets."
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Preparation of Keychain crednetials exfiltration"
```

Comment:
looks like a copy pasta from a previous rule

Reply:
indeed, changed to Kerberos Cached Crednetials Dumping